packages/account/package.json

```diff
@@ -28,6 +28,7 @@
     },
     "dependencies": {
         "@binary-com/binary-document-uploader": "^2.4.7",
+        "@deriv/api-types": "^1.0.54",
```

Comment:
Can we move this to devdependencies?

Reply:
I think it should be dev-depenendency . But in all other packages its listed under dependency

Reply:
Maybe we can move them to devdependencies later!